src/datajudge/db_access.py

```diff
@@ -902,3 +902,60 @@ def get_column_array_agg(
             for t in result
         ]
     return result, selections
+
+
+def get_ks_2sample(engine: sa.engine.Engine, table1: tuple, table2: tuple):
+    """
+    Runs the query for the two-sample Kolmogorov-Smirnov test and returns the test statistic d.
+    """
+    table1_selection, col1 = table1
+    table2_selection, col2 = table2
+
+    if is_mssql(engine):
+        table1_selection = str(table1_selection).replace('"', "")
+        table2_selection = str(table2_selection).replace('"', "")
+
+    ks_query_string = f"""
```

Comment:
The SQL here is fairly complicated so it would be great to have some explanation. I think you can just copy and paste your GH PR text.

Reply:
Is it okay for you if I just link to the full explanation here and give very short in-line comments in the code?